src/gmt_plot.c

```diff
@@ -8804,28 +8804,31 @@ struct PSL_CTRL *gmt_plotinit (struct GMT_CTRL *GMT, struct GMT_OPTION *options)
 			PSL_command (PSL, PSL_makecolor (PSL, GMT->current.setting.font_tag.fill.rgb));
 			PSL_command (PSL, " ");
 			PSL_setfont (PSL, GMT->current.setting.font_tag.id);
-			if (P->pen[0] || P->fill[0]) {	/* Must deal with textbox fill/outline */
+			if (P->pen[0] || P->fill[0] || P->shade[0]) {	/* Must deal with textbox fill/outline/shade */
+            /* All fills and pens have already gone through a syntax check in subplot -A */
```

Comment:
Sorry for being picky here, but the indentation on the new code makes this section difficult to read.

Reply:
Yes, I agree. Not sure why my Sublime Text is doing this.  I have the preferences set to Tab:

`"translate_tabs_to_spaces": false`

Not sure what to do about this - it is bugging me too.

Reply:
I use vs code so I do not know of a fix. I am on the branch now and could push a fix if you don't want to spend time figuring it out.